app/views/page_version/show.html.slim

```diff
@@ -70,35 +93,64 @@ table.diff-versions(data-fullheight='{ "bottom": 30, "cssrule": "min-height" }')
               - if @selected_version.page_version > @previous_version.page_version
                 h4(data-diff-title="original" data-empty=t('.untitled')) = @previous_version.title.presence || ""
                 div.table-responsive(data-diff-transcription="original" data-empty=t('.no_transcription_provided'))
-                  == xml_to_html(@previous_version.xml_transcription)
+                  div.source-code
+                    == @previous_version.transcription
+                  div.html-code
+                    == xml_to_html(@previous_version.xml_transcription)
                 - if @work.supports_translation
                   hr
                   h4= t('.translation')
                   div.table-responsive(data-diff-translation="original" data-empty=t('.no_translation_provided'))
-                    == xml_to_html(@previous_version.xml_translation)
+                    div.source-code
+                      == @previous_version.source_translation
+                    div.html-code
+                      == xml_to_html(@previous_version.xml_translation)
               - else
                 h4(data-diff-title="original" data-empty=t('.untitled')) = @selected_version.title.presence || t('.untitled')
                 div.table-responsive(data-diff-transcription="original" data-empty=t('.no_transcription_provided'))
-                  == xml_to_html(@selected_version.xml_transcription)
+                  div.source-code
+                    == @selected_version.transcription
+                  div.html-code
+                    == xml_to_html(@selected_version.xml_transcription)
                 - if @work.supports_translation
                   hr
                   h4= t('.translation')
                   div.table-responsive(data-diff-translation="original" data-empty=t('.no_translation_provided'))
-                    == xml_to_html(@selected_version.xml_translation)
+                    div.source-code
+                      == @selected_version.source_translation
+                    div.html-code
+                      == xml_to_html(@selected_version.xml_translation)
             - else
               h4(data-diff-title="original" data-empty=t('.untitled')) = @previous_version&.title.presence || ""
               div.table-responsive(data-diff-transcription="original" data-empty=t('.no_transcription_provided'))
-                == xml_to_html(@previous_version&.xml_transcription)
+                div.source-code
+                  == @previous_version.transcription
```

Comment:
Seems unhandled nil error here. Please do safe operation @previous_version&.transcription to avoid error when @previous_version is nil